scripts/submit_csv.sh

```diff
@@ -65,9 +89,44 @@ sed "
   s|%INPUT_FILES%|${INPUT_FILES}|g;
   s|%REQUIREMENTS%|${REQUIREMENTS}|g;
   s|%CSV_FILE%|${CSV_FILE}|g;
+  s|%ACCOUNT%|${ACCOUNT:-rrg-wdconinc}|g;
```

Comment:
Prefer to crash this if not specified, rather than have everyone submit under my allocation ;-)

Reply:
Example: `${ACCOUNT:?Define ACCOUNT with the slurm account}`, ref: https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion